tendrl/notifier/manager/__init__.py

```diff
@@ -0,0 +1,79 @@
+import etcd
+import gevent
+import signal
+
+from gevent.queue import Queue
+from tendrl.commons.event import Event
+from tendrl.commons.message import ExceptionMessage
+from tendrl.commons import TendrlNS
+from tendrl.commons.utils.log_utils import log
+from tendrl.notifier.notification import NotificationPluginManager
+from tendrl.notifier import NotifierNS
+
+
+class TendrlNotifierManager(object):
+    def __init__(self):
+        try:
+            NS.notification_queue = Queue()
```

Comment:
I think you are not using this remove if not used

Reply:
ah i missed this when i create new pull request :), yes it is not required, i will remove this